torch/_inductor/codegen/cpp.py

```diff
@@ -1555,6 +1555,23 @@ def is_load_only_block(self, sub_graph: torch.fx.Graph):
                 load_dtype = V.graph.get_dtype(name)
                 is_load_only = True
 
+            # Support "constant" node
+            if _node.target == "constant":
+                _, _, load_dtype = _node.args
+
+                # Create and record the context
+                opt_ctx = OptimizationContext()
+                opt_ctx.dtype = load_dtype
+                opt_ctx.ops_name = _node.target
+                _node.meta[OptimizationContext.key] = opt_ctx
```

Comment:
Should this be called from the caller (`masked`) side instead? They are duplicate and also `is_load_only_block` is supposed to be read-only?

Reply:
I prefer not to due to the complexity. If we call this within `masked`, we have to create another `CppVecKernelChecker` and prepare the context. The current `body` is not suitable for the invocation.